src/main/java/io/appium/java_client/HasDeviceDetails.java

```diff
@@ -0,0 +1,15 @@
+package io.appium.java_client;
+
+import static io.appium.java_client.MobileCommand.isKeyboardShownCommand;
+
+public interface HasDeviceDetails extends ExecutesMethod {
```

Comment:
I'd rather rename the interface to HasOnScreenKeyboard

Reply:
Hmm makes sense